app/presenters/hyrax/displays_image.rb

```diff
@@ -35,5 +37,9 @@ def iiif_endpoint(file_id)
           profile: Hyrax.config.iiif_image_compliance_level_uri
         )
       end
+
+      def image_format(channels)
+        channels.find { |c| c.include?('rgba') }.nil? ? 'jpg' : 'png'
```

Comment:
It looks like this will always return a jpg or png but do other IIIF image servers or IIIF clients make use of or expect anything different?  Maybe not.

Reply:
jpg is the only format required at all compliance levels, with png required for level 2. See https://iiif.io/api/image/2.1/compliance/#format